scripts/linters/html_linter.py

```diff
@@ -135,6 +137,29 @@ def handle_starttag(self, tag, attrs):
                             tag, line_number))
                     self.error_messages.append(error_message)
 
+                # Check if there are any white spaces around
+                # attribute assignment.
+                attr_val_structure = '{}="{}"' if value_in_quotes else '{}={}'
```

Comment:
Should we move the below code to a separate function _check_space_between_attributes_and_values? (The handle_starttag is already (v) big so moving it to a new function will make it readable.)

Reply:
sure will do it